src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -385,9 +389,14 @@ public RealJenkinsRule includeTestClasspathPlugins(boolean includeTestClasspathP
                     }
                     System.out.println("Will load plugins: " + Stream.of(plugins.list()).filter(n -> n.matches(".+[.][hj]p[il]")).sorted().collect(Collectors.joining(" ")));
                     base.evaluate();
+                } catch (Throwable t) {
+                    LOGGER.log(Level.WARNING, "Something went wrong", t);
+                    throw t;
                 } finally {
-                    if (proc != null) {
+                    try {
                         stopJenkins();
+                    } catch (Exception e) {
+                        LOGGER.log(Level.WARNING, "Jenkins was not stopped correctly", e);
                     }
```

Comment:
Why not simply

```suggestion
                    stopJenkins();
```

? If something failed, the test should fail. Why would you suppress this?

Reply:
agree, done in [aa5dd8f](https://github.com/jenkinsci/jenkins-test-harness/pull/465/commits/aa5dd8fcded997b2fe2d308fd7bb6d57c01d1b29)